docs/src/whatsnew/latest.rst

```diff
@@ -94,6 +94,12 @@ This document explains the changes made to Iris for this release
 #. `@pp-mo`_ updated to the latest CF Standard Names Table ``v78`` (21 Sept 2021).
    (:issue:`4479`, :pull:`4483`)
 
+#. `@SimonPeatman`_ added support for filenames in the form of a :class:`~pathlib.PurePath`
+   in :func:`~iris.load`, :func:`~iris.load_cube`, :func:`~iris.load_cubes`,
+   :func:`~iris.load_raw` and :func:`~iris.save` (:issue:`3411`, :pull:`3917`).
+   Support for :class:`~pathlib.PurePath` is yet to be implemented across the rest
+   of Iris (:issue:`4523`).
```

Comment:
```suggestion
   Note that, full support for :class:`~pathlib.PurePath` throughout Iris has yet to be implemented.
   (:issue:`4523`).
```

Reply:
@wjbenfold Do we have a follow-up issue to state that we need to do this?

Reply:
#4523